docs/source/whatsnew/pr/restore-line-numbers.rst

```diff
@@ -0,0 +1,5 @@
+Restore line numbers for Input
+==================================
+
+Line number information in tracebacks from input are restored.
+Line numbers from input were removed during the transition to v8 enhanced traceback reporting.
```

Comment:
It would be super nice discoverability-wise to include an example of changed traceback here. Just so that the users see that _we care_ ‍😁.

Reply:
Done.